hyperstart.go

```diff
@@ -43,10 +47,11 @@ const (
 // HyperConfig is a structure storing information needed for
 // hyperstart agent initialization.
 type HyperConfig struct {
-	SockCtlName string
-	SockTtyName string
-	Volumes     []Volume
-	Sockets     []Socket
+	SockCtlName  string
+	SockTtyName  string
+	Volumes      []Volume
+	Sockets      []Socket
+	PauseBinPath string
```

Comment:
The whole pause (container, binary, path, etc...) should be fully transparent to the caller.
In other words, we (The virtcontainers build system) should build it and install it, and the caller should not handle any of this. It probably means that we want to have a simple Makefile for building and installing the pause binary and to build the library itself.